geoprocessing/src/filters/all-exceptions.exception.filter.ts

```diff
@@ -67,11 +67,11 @@ export class AllExceptionsFilter implements ExceptionFilter {
         ? omit(errorData, ['meta.rawError', 'meta.stack'])
         : errorData,
     );
-
+    response.removeHeader('content-encoding');
     response
       .status(status)
       .header('Content-Type', 'application/json')
-      .header('Content-Disposition', 'inline')
+      .header('content-disposition', 'inline')
```

Comment:
not sure if the uppercase affects here (e.g 'Content-disposition vs content-disposition). We have this in uppercase in all the controllers endpoints.

Reply:
I found this change in the commits of the old branch (was authored by Alicia IIRC), in practice headers are case-insensitive so I left it as it was, but I'll change to stick to our usual convention.

Reply:
fixed in 857d81a86f28224b0bbb737ebd979bdafb37698d